### ISSUE TITLE
`switchInput` ignoring `size` parameter in version 0.6.3

### ISSUE THREAD

**Issue by Mosk915** [dreamRs/shinyWidgets#472]
I'm using the latest version of `shinyWidgets` (0.6.3) and when I create a `switchInput`, the `size` argument is ignored.  I verified that this is not an issue on version 0.6.2.

```r
library(shiny)
library(shinyWidgets)

ui <- fluidPage(
  switchInput(inputId = "test", size = "mini")
)

server <- function(input, output, session) {
  
}

shinyApp(ui, server)
```
0.6.2:
![image](https://user-images.githubusercontent.com/59397359/152562896-c0af1404-c735-490b-8756-9e5cc0be4c8d.png)

0.6.3:
![image](https://user-images.githubusercontent.com/59397359/152562706-e2f00eac-04ff-4572-938f-02a65d20a927.png)

**Comment by pvictor** [comment]
It should be fixed in version 0.6.4, now on CRAN.

**Comment by pvictor** [comment]
thanks for the confirmation!
